docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -631,13 +631,13 @@ and
 Coverage Propensities
 ++++++++++++++++++++++
 
-.. todo::
+The coverage propensity for wasting treatment parameters :math:`C` and :math:`E` for any given simulant should update upon each transition between wasting states. 
```

Comment:
This means draw a new propensity from a independent uniform distribution, right?